docs/docs/external/getting-started/interop/asset-transfer.md

```diff
@@ -126,3 +128,177 @@ Assuming that the `simpleassettransfer` chaincode has been deployed in both netw
    ```bash
    ./scripts/getTokenBalance.sh network2 bob
    ```
+
+## 2. Corda with Corda
+One Corda network transfers either a bond or some tokens owned by `PartyA` (`CORDA_PORT=10006`) to `PartyB` (`CORDA_PORT=30009`) in the other network.
+
+### Transfer or recover token (fungible) assets
+Assume that the corDapp `cordaSimpleApplication` has been deployed in both networks.
+- Navigate to `samples/corda/corda-simple-application` folder.
+- Create a `network-id` for each Corda network. This is a network state, and will be available in the vault of all the parties which are members of the network (if required, run the command `./clients/build/install/clients/bin/clients util get-party-name` with `CORDA_PORT=10006` or `CORDA_PORT=10009` or `CORDA_PORT=30006` or `CORDA_PORT=30009` to fetch the name of the parties `PartyA` in `Corda_Network` or `PartyB` in `Corda_Network` or `PartyA` in `Corda_Network2` or `PartyB` in `Corda_Network2` respectively).
```

Comment:
I'd recomment to move this step to Ledge Initialization page, as we want this page to totally focus on Asset Transfer commands only, all initialization should be done before landing on this page.

Reply:
Done.